app/src/main/res/layout/activity_main.xml

```diff
@@ -1,17 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
-    
-    <FrameLayout
-        android:id="@+id/fragment_products"
-        android:layout_width="wrap_content"
-        android:layout_height="match_parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        ></FrameLayout>
-</android.support.constraint.ConstraintLayout>
\ No newline at end of file
+    tools:context=".MainActivity"
+    android:orientation="horizontal">
+
+    <LinearLayout
+        android:layout_weight=".6"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+        <Button
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:text="test"/>
+
+    </LinearLayout>
+    <LinearLayout
+        android:layout_weight=".3"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+        <FrameLayout
+            android:id="@+id/main_activity_fragment_products"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+    </LinearLayout>
```

Comment:
umm, i think you can shorten these codes by removing LinearLayout tag and move the weight attribute into FrameLayout tag

Reply:
I already shorten what you ask. And i add another file since I forgot to add it